NativePlugin/code/build.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: 'com.android.library'
+
+android {
+  compileSdkVersion 25
+  buildToolsVersion '25.0.2'
+
+  defaultConfig {
+    externalNativeBuild {
+      cmake {
+        targets 'Earcut'
+        cppFlags '-std=c++11'
+        arguments '-DANDROID_TOOLCHAIN=clang',
+                  '-DANDROID_STL=c++_shared'
+                    cppFlags '-std=c++14',
+                  '-pedantic',
+                  '-fPIC'
+      }
+    }
+  }
+
+  externalNativeBuild {
+    cmake {
+      path '../CMakeLists.txt'
+    }
+  }
+
+  buildTypes {
+    release {
+      externalNativeBuild {
+        cmake.cppFlags'-g0'
+      }
+    }
+  }
+
+  productFlavors {
+    full {
+      externalNativeBuild.cmake.abiFilters 'armeabi-v7a', 'x86'
+    }
+  }
+}
+
+task installLibraries(type: Copy)
+
+installLibraries {
+  description = 'Install libraries into the Unity plugin project...'
+  from 'build/outputs/aar/'
+  into '../../Assets/Plugins/Android/'
+  include 'code-full-release.aar'
+  rename 'code-full-release.aar', 'Earcut.aar'
```

Comment:
Perfect, I'll add a note on the native plugin build step README to make sure it is not an issue in the future.